src/scripts/inclusion-bot.test.js

```diff
@@ -63,33 +64,38 @@ describe("Inclusion bot", () => {
     },
   };
 
-  beforeAll(() => {
-    bot.getTriggers = () => ({
-      link: "http://link.url",
-      message: "This is the message",
-      triggers: [
-        {
-          alternatives: ["a1", "a2", "a3"],
-          ignore: ["not match 1"],
-          matches: /(match 1)/i,
-        },
-        {
-          alternatives: ["b1"],
-          matches: /(match 2a|match 2b)/i,
-        },
-      ],
-    });
-  });
-
   beforeEach(() => {
     jest.resetAllMocks();
+
+    fs.readFileSync.mockReturnValue(`
+link: https://link.url
+message: "This is the message"
+
+triggers:
+  - matches:
+      - match 1
+    alternatives:
+      - a1
+      - a2
+      - a3
+    ignore:
+      - not match 1
+  - matches:
+      - match 2a
+      - match 2b
+    alternatives:
+      - b1
+`);
```

Comment:
I realized it made more sense _*not*_ to mock the internal behavior and instead mock the external data. The internal processing of the file is already tested further up, but I'm not sure that's proper either. In any case, this lets the internal behavior change without having to change the test mocks too.